l10n_it_intrastat/models/account.py

```diff
@@ -268,11 +269,10 @@ def _prepare_intrastat_line_amount(self, res):
         amount_currency = self.price_subtotal
         company_currency = self.invoice_id.company_id.currency_id
         invoice_currency = self.invoice_id.currency_id
-        amount_euro = invoice_currency._convert(
+        amount_euro = invoice_currency.compute(
             amount_currency,
             company_currency,
-            self.invoice_id.company_id,
-            fields.Date.today())
+            self.invoice_id.company_id)
```

Comment:
come terzo parametro del metodo c'è `round` non `company_id`: così facendo passi `round=True` vedi [qui](https://github.com/odoo/odoo/blob/91807404e7d5c314ea30926b44bd16e878417ec1/odoo/addons/base/res/res_currency.py#L150) che è sì il default ma va da sé che non è corretto passarlo in questo modo!

Reply:
Grazie della review! Hai ragione, ho corretto